main.py

```diff
@@ -0,0 +1,23 @@
+from sanic import Sanic
+from sanic import response
+from utils import perform_notebook_conversion, authorized
+
+
+# Webservice routing
+app = Sanic('douglass')
+app.config.from_pyfile('config.py')
```

Comment:
how is the application going to pick the right config file based on the environment it's running in?

Reply:
I'm hoping to get some advice on this from the Terra UI team. They wrote a deploy script that copied one of the environment examples from config.ENV.py to config.py I will probably do something similar